components/CMakeLists.txt

```diff
@@ -236,6 +236,23 @@ if (UNIX AND NOT APPLE)
 target_link_libraries(components ${CMAKE_THREAD_LIBS_INIT})
 endif()
 
+add_library(osgdb_serializers_nifosg SHARED
+    serializers/nifloader serializers/controller
+    serializers/particle serializers/userdata
+    )
+
+target_link_libraries(osgdb_serializers_nifosg components)
+
+add_library(osgdb_serializers_sceneutil SHARED
```

Comment:
I assume so, but I have not been able to figure this out yet.  Perhaps there's a "feature" in the OSG serializer code such that everything only compiles and runs correctly when the namespaces all match correctly.  This wouldn't be shocking given th number of macros involved, and the duplication of very similar values.  I'll try to play some tricks to get us into one openmw dynamic serializer library without changing any existing code.